presto-plugin-toolkit/src/test/java/io/prestosql/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -268,4 +314,14 @@ private String getResourcePath(String resourceName)
     {
         return this.getClass().getClassLoader().getResource(resourceName).getPath();
     }
+
+    private static void assertDenied(ThrowingRunnable runnable)
+    {
+        assertThrows(AccessDeniedException.class, runnable);
```

Comment:
can you use assertj `assertThatThrownBy()`? Please also verify error message (it is quite important to know that the proper exception was raised).

Reply:
@kokosing - This is done